composer.json

```diff
@@ -67,6 +70,11 @@
         "sonata-project/media-bundle": "^3.20",
         "symfony/browser-kit": "^4.4 || ^5.1",
         "symfony/phpunit-bridge": "^5.1.8",
+        "symfony/routing": "^4.4",
+        "symfony/security-bundle": "^4.4",
+        "symfony/security-csrf": "^4.4",
+        "symfony/translation": "^4.4",
+        "symfony/twig-bundle": "^4.4",
```

Comment:
why as require-dev?

Reply:
I guess it's because it's needed for functional tests only

Reply:
>I guess it's because it's needed for functional tests only

That's right.